Functional Monitoring documentation update (4 of 5) (#53)
mulesoft/docs-functional-monitoring: modules/ROOT/pages/afm-monitoring-private-apis.adoc

====

== Association of Private Locations with Schedules

> The tests in monitors are run periodically according to a selected schedule. For a private location to run according to a schedule, a schedule must be associated with it. A private location can be associated with more than one schedule for more than one monitor.

Reviewer: This seems overly wordy, and it begs the question of public (?) location monitor testing vs. private?
Why are we specifying "private" here; don't _all_ locations need association with a schedule to run tests?
Also, this para goes from tests running according to schedule, to locations running according to schedule, which does not sound correct: this is about the test + schedule, right?

Reviewer: My best guess:
```suggestion
Because monitor tests run according to specified schedules, monitoring your private location requires that location to be associated with one or more schedules (one per monitor). 
```

Author: Hmm, good points. I'm now thinking that the paragraph can simply be removed, as it's not presenting any new information.

== Update Private Locations

> As updates become available, you'll receive notifications in the *Locations* modal dialog. Follow the prompts to apply the update to all private locations.

Reviewer: Avoid future tense:
```suggestion
As updates become available, you are notified in the *Locations* modal dialog. Follow the prompts to apply the update to all private locations.
```
